src/interactions/forbidClickInteraction.ts

```diff
@@ -30,6 +30,7 @@ class ForbidenClickInteraction extends AbstractInteraction {
   }
 
   async pipe(event: InteractionEvent) {
+    if (!this.active) return event
```

Comment:
Just a question here: by default even if the interaction is disabled, the pipe function can be called? Or was this line added as a security issue?